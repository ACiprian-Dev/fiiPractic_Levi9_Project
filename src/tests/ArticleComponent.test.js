import {mount, shallow} from 'enzyme';
import ArticleComponent from '../components/ArticleComponent';
import toJson from 'enzyme-to-json';

describe('ArticleComponent test', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(<ArticleComponent title="prim"></ArticleComponent>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
    it('should render title', () => {
        const wrapper = shallow(<ArticleComponent title="ceva"></ArticleComponent>);
        expect(wrapper.find(".articleTitle").text()).toBe("ceva");
    })
    it('should mount ArticleComponent', async () => {
        const wrapper = await mount(<ArticleComponent title="ceva"/>)
        expect(wrapper.find(".articleTitle").text()).toBe("ceva")
    })
})