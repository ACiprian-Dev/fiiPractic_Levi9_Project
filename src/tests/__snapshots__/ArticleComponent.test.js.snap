// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleComponent test should match snapshot 1`] = `
<div
  className="articleContainer"
>
  <div
    className="articleTitle"
  />
  <div
    className="articleContent"
  />
</div>
`;
